dev/tools/gen_defaults/lib/template.dart

```diff
@@ -202,7 +203,21 @@ abstract class TokenTemplate {
       return 'null';
     }
     final String borderColor = componentColor(componentToken);
-    final double width = (tokens['$componentToken.width'] ?? 1.0) as double;
+    final double width = (tokens['$componentToken.width'] ?? tokens['$componentToken.height'] ?? 1.0) as double;
+    return 'BorderSide(color: $borderColor${width != 1.0 ? ", width: $width" : ""})';
+  }
+
+  /// Generate a [BorderSide] for the given components.
+  String mergedBorder(String componentToken1, String componentToken2) {
```

Comment:
I am a little confused as to what this is doing? It looks like it is creating a border from either of the components' values with the first component taking priority. I don't completely understand the use case for this. Also, is this something that we think will be generally useful for other templates or something that is just specific to the InputDecorator? It might make sense to make it a method in just that template.

Reply:
Yes this is only for InputDecorator, because the text field "error focus" border should be 2 dp but the token file doesn't have a 2 dp height for it. So I am trying to merge the "error focus" border style with the "focus" border style for this case.

You are right, it make more sense to move it to just that template. 
moved it to `input_decorator_template.dart`